project-base/app/assets/js/admin/grapesjs/locales/en.js

```diff
@@ -0,0 +1,190 @@
+export const cs = {
```

Comment:
I suppose the const name should be `en`, right?

Reply:
Fixed in forgotten commit